### ISSUE TITLE
Name change for the sake of browserify compatibility

### ISSUE THREAD

**Issue by alexgorbatchev** [theakman2/node-modules-dustify#1]
Hey friend, I stumbled on this module because I was searching for a [Browserify](https://github.com/substack/node-browserify) transformation plugin. If you aren't familiar, [Browserify](https://github.com/substack/node-browserify) is a very popular module that allows you to bundle up all your javascript files for distribution to the client using `require`.

[Browserify](https://github.com/substack/node-browserify) has a [list of transforms](https://github.com/substack/node-browserify/wiki/list-of-transforms) that follow the "something-iffy" naming pattern. 

I noticed that this module isn't actively maintained or used. Would you be in willing to give up NPM name so that we can rename [dustjs-browserify](https://github.com/scottbrady/dustjs-browserify) to `dustify`?

**Comment by alexgorbatchev** [comment]
@theakman2  ping

**Comment by theakman2** [comment]
Sure, I don't mind giving up the name. How do you give up a name exactly?

**Comment by theakman2** [comment]
Date: Fri, 1 May 2015 08:05:01 -0700

**Comment by alexgorbatchev** [comment]
Amazing, thank you! Add me as an owner via `npm owner add alexgorbatchev` command and I can take over publishing the other module. Then simply change `name` in your package.json to something else and republish with a new name!

Thanks again!

**Comment by alexgorbatchev** [comment]
@theakman2  ping!

**Comment by alexgorbatchev** [comment]
@theakman2 :rocket:

**Comment by alexgorbatchev** [comment]
hey friend? you around?

**Comment by theakman2** [comment]
Yes, I added you as an owner and changed the name of my package a little while ago.

Date: Mon, 11 May 2015 17:05:58 -0700

**Comment by alexgorbatchev** [comment]
Thank you very much! Really appreciate it!
